packages/ubuntu_desktop_installer/lib/pages/active_directory/active_directory_l10n.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:subiquity_client/subiquity_client.dart';
+
+import '../../l10n.dart';
+
+extension AdDomainNameValidationL10n on AdDomainNameValidation {
+  String localize(AppLocalizations lang) {
+    switch (this) {
+      case AdDomainNameValidation.OK:
+        return '';
+      case AdDomainNameValidation.EMPTY:
+        return lang.activeDirectoryDomainEmpty;
+      case AdDomainNameValidation.TOO_LONG:
+        return lang.activeDirectoryDomainTooLong;
+      case AdDomainNameValidation.INVALID_CHARS:
+        return lang.activeDirectoryDomainInvalidChars;
+      case AdDomainNameValidation.START_DOT:
+        return lang.activeDirectoryDomainStartDot;
+      case AdDomainNameValidation.END_DOT:
+        return lang.activeDirectoryDomainEndDot;
+      case AdDomainNameValidation.START_HYPHEN:
+        return lang.activeDirectoryDomainStartHyphen;
+      case AdDomainNameValidation.END_HYPHEN:
+        return lang.activeDirectoryDomainEndHyphen;
+      case AdDomainNameValidation.MULTIPLE_DOTS:
+        return lang.activeDirectoryDomainMultipleDots;
+      default:
+        throw UnimplementedError(name);
```

Comment:
missing translation for REALM_NOT_FOUND
![image](https://user-images.githubusercontent.com/140617/222716652-f6556405-6a7f-4ba2-becd-ebdc516ef604.png)

Reply:
![image](https://user-images.githubusercontent.com/11138291/222718874-8dfeeb65-bc11-4280-a3a8-ed7c865ac831.png)

versus

![image](https://user-images.githubusercontent.com/11138291/222718822-ba2139ef-db7d-431e-95b0-d24721362016.png)

Reply:
Honestly: much better than the alternatives.